src/agent/README.md

```diff
@@ -43,10 +43,9 @@ The `rust-agent` depends on [`grpc-rs`](https://github.com/pingcap/grpc-rs) by P
 - https://github.com/alipay/grpc-rs/tree/rust_agent
 
 ### Build from Source
-The rust-agent need to be built with rust nightly, and static linked with musl.
+The rust-agent need to be built with rust newer than 1.39, and static linked with musl.
```

Comment:
Is this change related to the gRPC change below? If not, strictly it should be in a different commit with some details as to why the new minimum version is 1.39.